llvm/test/CodeGen/RISCV/macro-fusions-veyron-v1.mir

```diff
@@ -0,0 +1,159 @@
+# REQUIRES: asserts
```

Comment:
Do we need end-to-end llc tests like `llvm/test/CodeGen/RISCV/macro-fusion-lui-addi.ll`?